src/plugins/dataRefs/dataRefs.ts

```diff
@@ -176,16 +84,16 @@ const DataRef = (opst: DataRefsPluginOptions) => {
             calcRefs: (fromIdx: number, toIdx: number, seriesId: string) => {
                 const seriesIdx = _.state.y2uIdx[seriesId];
                 const timestamps = _.uplot.data[0].slice(fromIdx, toIdx + 1) as number[];
-                const values = _.uplot.series[seriesIdx].$c.slice(fromIdx, toIdx + 1) as (number | null)[];
+                const values = _.uplot.series[seriesIdx].$c;
```

Comment:
Would you kindly give $c a human readable name?

As well as I see it's  $c: rawSeries.data,

Reply:
I will be a breaking change. Planning for v4